digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -291,6 +293,19 @@ class TrafficSignUpdateAssets(trafficSignService: TrafficSignService, trafficSig
   }
 }
 
+class LaneWorkListInsertItem(laneWorkListService: LaneWorkListService) extends Actor {
+  def receive = {
+    case x: (String, Option[Int],LinkProperties, VVHRoadlink, Option[String]) =>
```

Comment:
en oikein tykkään näin monimutkaisesta tuple. case class olis parempi

Reply:
Oma ajatus tässä oli painottaa RoadLinkServicen säilyttämistä siistinä

Reply:
On kyllä selvempi jos käärin vaan case class